plugins/virt.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: virt
+spec:
+  version: "v0.17.2"
+  platforms:
+    - selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+      uri: "https://github.com/dhiller/kubectl-virt-plugin/releases/download/v0.17.2/virtctl-v0.17.2-linux-amd64.tar.gz"
+      sha256: "bbf3b4f15b64ffe4c9196b53d045a92bc8113d24cfde471dded3c8c505aef663"
+      files:
+        - from: "/virtctl/virtctl-v0.17.2-linux-amd64"
+          to: "."
+      bin: "./virtctl-v0.17.2-linux-amd64"
+    - selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+      uri: "https://github.com/dhiller/kubectl-virt-plugin/releases/download/v0.17.2/virtctl-v0.17.2-windows-amd64.exe.tar.gz"
+      sha256: "828876ae8abc50c7965f7b9eaa1ff3f6d3d9b0dad25c79e6a7d49dd6a1543087"
+      files:
+        - from: "/virtctl/virtctl-v0.17.2-windows-amd64.exe"
+          to: "."
```

Comment:
When you have version names in your binary, consider renaming here so you don’t update the bin field every time on a version change.

Reply:
+1 for renaming. This should work:
```yaml
- from: "/virtctl/virtctl-v0.17.2-windows-amd64.exe"
  to: "virtctl.exe"
  bin: "virtctl.exe"
```